amazon-kinesis-client-multilang/pom.xml

```diff
@@ -21,7 +21,7 @@
   <parent>
     <artifactId>amazon-kinesis-client-pom</artifactId>
     <groupId>software.amazon.kinesis</groupId>
-    <version>2.4.5</version>
+    <version>2.4.6-SNAPSHOT</version>
```

Comment:
if you plan to release this as snapshot version first for now, i think this should be on another branch in remote repo ( so that master is always open for new urgent releases). 


Regardless, release on master branch should have release documentation updates? I am assuming because this was snapshot release its not added yet?

Reply:
> i think this should be on another branch in remote repo ( so that master is always open for new urgent releases).

Absolutely agreed on the second half (and this PR is ready for release, assuming no reviewer dissent). I'll reach out directly so it's faster+easier for us to discuss the 2.4.6 release strategy.

Reply:
FYI: I will submit a separate PR w/ the doc updates (e.g., changelog) that is inclusive of our known PRs. This may help minimize merge conflicts as we parallelize.